src/libraries/System.Runtime/src/System.Runtime.csproj

```diff
@@ -1,11 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <ContractProject>$(LibrariesProjectRoot)System.Runtime.Experimental\ref\System.Runtime.Experimental.csproj</ContractProject>
```

Comment:
Probably worth a comment here about what this is doing.

Reply:
Added `<!-- Pull type forwards from the experimental ref assembly so we only need to maintain a single src project -->`